snippets/csharp/VS_Snippets_Misc/tpl_parallel/cs/parallelinvoke.cs

```diff
@@ -100,31 +98,28 @@ static string[] CreateWordArray(string uri)
         }
         #endregion
     }
-
-    /* Output (May vary on each execution):
-        Retrieving from http://www.gutenberg.org/dirs/etext99/otoos610.txt
-        Response stream received.
-        Begin first task...
-        Begin second task...
-        Task 2 -- The most common words are:
-          species
-          selection
-          varieties
-          natural
-          animals
-          between
-          different
-          distinct
-          several
-          conditions
-
-        Begin third task...
-        Task 1 -- The longest word is characteristically
-        Task 3 -- The word "species" occurs 1927 times.
-        Returned from Parallel.Invoke
-        Press any key to exit  
-     */
 }
+//        The example displays output like the following:
+//              Retrieving from http://www.gutenberg.org/files/54700/54700-0.txt
+//              Begin first task...
+//              Begin second task...
+//              Begin third task...
+//              Task 2 -- The most common words are:
+//              Oblomov
+//              himself
+//              Schtoltz
+//              Gutenberg
+//              Project
+//              another
+//              thought
+//              Oblomov's
+//              nothing
+//              replied
+//       
+//              Task 1 -- The longest word is incomprehensible.
+//              Task 3 -- The word "sleep" occurs 57 times.
+//              Returned from Parallel.Invoke
+//              Press any key to exit
 // </snippet06>
 
 
```

Comment:
should you change the word from species to sleep in lines 152 and 140 as well? Or maybe delete that part of the code if it's not used anywhere?